_dashboards/get-started/dql.md

```diff
@@ -0,0 +1,178 @@
+
+```
+host.keyword:*.example.com/*
+```
+{% include copy.html %}
+
+To check whether a field exists or has any data, use a wildcard to see whether Dashboards returns any results:
+
+```
+host.keyword:*
+```
+{% include copy.html %}
+
+## Searching with Boolean queries
+
+To mix and match or combine multiple queries for more refined results, you can use the Boolean operators `and`, `or`, and `not`. DQL is not case sensitive, so `AND` and `and` are the same.
+
+```
+host.keyword:www.example.com and response.keyword:200
+```
+{% include copy.html %}
+
+The following example shows how to use multiple operators in one query:
+
+```
+geo.dest:US or response.keyword:200 and host.keyword:www.example.com
+```
+{% include copy.html %}
+
+Remember that Boolean operators follow the logical precedence order of `not`, `and`, and `or`, so if you have an expression like the one in the preceding example, `response.keyword:200 and host.keyword:www.example.com` is evaluated first.
+
+To avoid confusion, use parentheses to dictate the order in which you want to evaluate operands. If you want to evaluate `geo.dest:US or response.keyword:200` first, use the following expression:
+
+```
+(geo.dest:US or response.keyword:200) and host.keyword:www.example.com
+```
+{% include copy.html %}
+
+## Querying dates and ranges
+
+DQL supports numeric inequalities, for example, `bytes >= 15 and memory < 15`.
+
+You can use the same method to find a date before or after the date specified in the query. `>` indicates a search for a date after the specified date, and `<` returns dates before the specified date, for example, `@timestamp > "2020-12-14T09:35:33`.
+
+## Querying nested fields
+
+Searching a document with [nested fields]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/nested/) requires you to specify the full path of the field to be retrieved. In this example, the `superheroes` field has nested objects.
+
+```json
+{
+ "superheroes":[
+    {
+      "hero-name": "Superman",
+      "real-identity": "Clark Kent",
+      "age": 28
+    },
+    {
+      "hero-name": "Batman",
+      "real-identity": "Bruce Wayne",
+      "age": 26
+    },
+    {
+      "hero-name": "Flash",
+      "real-identity": "Barry Allen",
+      "age": 28
+    },
+    {
+      "hero-name": "Robin",
+      "real-identity": "Dick Grayson",
+      "age": 15
+    }
+  ]
+}
+```
+{% include copy.html %}
+
+To retrieve a specific field using DQL, use the following notation:
+
+```
+superheroes: {hero-name: Superman}
+```
+{% include copy.html %}
+
+To retrieve multiple objects from your document, specify all the fields you want to retrieve, as shown in the following notation:
+
+```
+superheroes: {hero-name: Superman} and superheroes: {hero-name: Batman}
+```
+{% include copy.html %}
+
+The previous Boolean and range queries still work, so you can submit a more refined query, as shown in the following notation:
```

Comment:
Same comment re: what we mean by "still work."

Reply:
Revised. "Continuing with the preceding Boolean and range queries, you can..."

Reply:
Revised. "Continuing with the preceding Boolean and range queries, you can submit a more refined query, for example:"